Consul + Vault - TLS (#195)
hashicorp/katakoda: consul-secure-tls-vault/step3.md

> Once configuration is created and certificates and key are in place you can start Consul.

Reviewer: ```suggestion
Once the agent configuration is created and thee certificates are distributed, you can start Consul.
```

First create Consul data directory

`mkdir -p /opt/consul/data`{{execute T1}}

`consul agent \
     -config-file server.json \
     -config-file server-tls.json \
     -advertise '{{ GetInterfaceIP "ens3" }}' \
     -data-dir=/opt/consul/data`{{execute T1}}

> In the output, if the configuration was successful,

Reviewer: ```suggestion
In the output
```

> you will get an indication in the output that TLS encryption is now enabled:

Reviewer: ```suggestion
The output will indicated that TLS encryption is now enabled:
```

```
...
==> Starting Consul agent...
           Version: '1.8.3'
           Node ID: 'db8dd611-e833-e50e-3a5b-fc21fd63c348'
         Node name: 'server1'
        Datacenter: 'dc1' (Segment: '<all>')
            Server: true (Bootstrap: true)
       Client Addr: [127.0.0.1] (HTTP: 8500, HTTPS: 8501, gRPC: -1, DNS: 8600)
      Cluster Addr: 172.17.0.32 (LAN: 8301, WAN: 8302)
           Encrypt: Gossip: false, TLS-Outgoing: true, TLS-Incoming: true, Auto-Encrypt-TLS: true

==> Log data will now stream in as it occurs:
...
```

> At this point in the lab you have a Consul server with TLS encryption configured.

Reviewer: ```suggestion
At this point, you have a Consul server with TLS encryption configured.
```
